partner_industry_pricelist/models/res_partner_industry.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class ResPartnerIndustry(models.Model):
+
+    _inherit = 'res.partner.industry'
+
+    pricelist_id = fields.Many2one(
+        string='Pricelist',
+        comodel_name='product.pricelist',
+    )
```

Comment:
we could have a button @button_apply_pricelist() to set the pricelist to all partner that have a given ``industry_id``, don't you think ?